public/client.js

```diff
@@ -98,18 +98,39 @@ socket.on('Display Message', function(data) {
     else {
         class_name = 'others'
     }
+    // Adding Emoji
+    var p = data.msg;
+    var colon1 = p.indexOf(":");
+    while (colon1 != -1) {
+      var colon2 = p.indexOf(":", colon1 + 1);
+      if (colon2 != -1) {
+        emoji = p.slice(colon1+1, colon2);
+        if (emojiList.includes(emoji)){
+          var p = p.slice(0, colon1) + "<img class=\"emoji\" src=\"images/emoji/" + emoji + ".png\">" + p.slice(colon2 + 1);
+          colon1 = p.indexOf(":", colon2 + 1);
+        }
+        else {
+          break
```

Comment:
I don't think this should be a break here.
Otherwise `:notanemoji: followed by :emoji:` will not work

Reply:
I will again look into that and will find a better solution.

Reply:
I fixed this in 1deea3d.